.gitignore

```
@@ -47,3 +47,4 @@ google-services.json
 # OS specific
 .DS_Store
 *.log
+/.idea/deploymentTargetDropDown.xml
```

Обрежьте до 
```suggestion
/.idea
```
чтобы все файлы из папки не попадали в репозиторий, они лишние